doc/nrf/ug_thread_commissioning.rst

```diff
@@ -80,6 +80,25 @@ Minimal Thread Device (MTD)
 
 For more information, see `Device Types on OpenThread portal`_.
 
+SED vs SSED activity
+--------------------
+
+.. thread_ssed_description_start
+
+A Thread Synchronized Sleepy End Device is synchronized with its parent router and uses the radio only at scheduled intervals, by using the Coordinated Sampled Listening (CSL) feature introduced in the Thread v1.2.
+During those intervals, the device waits for the router to send it any data related to the desired device activity.
+The SSED device does require in fact sending packets every now and then to keep synchronization with the router.
+However, unlike the regular SED device, the SSED does not actively communicate with the router by polling and goes into the idle mode in-between the scheduled activity periods.
+In case of the absence of application-related traffic for a longer period of time, the SSED sends data poll request packet to synchronize with the parent.
```

Comment:
```suggestion
In case of there is no application-related traffic for a longer period of time, the SSED sends a data poll request packet to synchronize with the parent.
```